sdks/python/apache_beam/utils/annotations.py

```diff
@@ -69,7 +69,7 @@ def exp_multiply(arg1, arg2):
 warnings.simplefilter("once")
 
 
-def annotate(label, since, current):
+def annotate(label, since, current, extra_message=None):
```

Comment:
Why `extra_message` defaults to `None`, other arguments do not. They are optional since the definition of `deprecated` and `experimental` passes these values. They can do the same for `extra_message`.